packages/apimonitor.vm/apimonitor.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>apimonitor.vm</id>
+    <version>2.13.0.20210213</version>
```

Comment:
[nitpick] The forth segment should use today's date according to [our documentation](https://github.com/mandiant/VM-Packages/wiki/Contributing#version):
> For packages that install tools via URLs and use three or fewer version segments, use the tool's version string. For tools that use four version segments, use the tool's version segments one through three and the current date in the format `0.0.0.YYYYMMDD` for the fourth segment.

But as long as the date is increased when modifying the package (what is easy with any date) it doesn't matter much. So I am fine with leaving it as it is. But I wanted to point this out.

Reply:
Gotcha, I copied this from the original package.

Reply:
@Ana06  see my comment regarding this, since this is a meta-package.